modules/oadp-release-notes-1-2-3.adoc

```diff
@@ -34,5 +34,10 @@ For a complete list of all issues resolved in the release of OADP 1.2.3, see the
 [id="known-issues-1-2-3_{context}"]
 == Known issues
 
-There are no known issues in the release of OADP 1.2.3.
+The {oadp-short} 1.2.3 has the following known issue:
 
+.DPA does not reconcile when the credentials secret is updated
+
+There are no {oadp-short} specific labels or owner references on the `cloud-credentials` secret. Therefore, the Operator does not reconcile when you update the `cloud-credentials` secret. This means you could create a `cloud-credentials` secret with bad credentials, empty data for example, then the Operator will reconcile and create Backup Storage Location (BSL) and registry deployment with bad credentials. However, when you update  `cloud-credentials` secret with _good_  credentials, the Operator does not immediately reconcile to catch the new credentials. This issue is resolved in {oadp-short} 1.3.
```

Comment:
```suggestion
Currently, the Operator does not reconcile when you update the `cloud-credentials` secret. This occurs because there are no {oadp-short} specific labels or owner references on the `cloud-credentials` secret. If you create a `cloud-credentials` secret with incorrect credentials, such as empty data, the Operator reconciles and creates a Backup Storage Location (BSL) and registry deployment with the empty data. As a result, when you update  `cloud-credentials` secret with the correct credentials, the Operator does not immediately reconcile to catch the new credentials. This issue is fixed in {oadp-short} 1.3.
```

Reply:
I had a hard time connecting the customer experience to the cause of the known issue. I find that rearranging the order of the sentences makes it to easier to follow. Does this change the intended meaning too much?

Also, if there is no workaround for this issue in OADP 1.2.3, I think it is worth explicitly stating that the workaround is to update to 1.3. It is implied in what you have here, but I think this would benefit from being explicit.

The change from bad/good to incorrect/correct is not a blocker, it just sounds a little better to my ear. The same with removing the italicized good/correct. I think that the meaning is already clear to the reader.

Calling out the [IBM Style rule](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) to avoid future tense.